content/vendor/cli/installer create.md

```diff
@@ -0,0 +1,67 @@
+---
+date: 2021-09-10
+linktitle: "installer create"
+title: "installer create"
+weight: 90115
+---
+
+Create a new installer spec by providing YAML configuration for a https://kurl.sh cluster.
```

Comment:
Create a new installer spec by providing a YAML configuration file for a https://kurl.sh cluster.